From 91f351705c9e7b8d8754bfda485d59c9e4d0461f Mon Sep 17 00:00:00 2001
From: henning <>
Date: Tue, 21 Apr 2026 14:36:00 +0000
Subject: [PATCH 04/30] newer gcc is so smart to point out that
 settime_deadline may be used uninitialized. of course it is wrong. sprinkle a
 "= 0" to shut it up. pointed out by bcook, discussed with, gcc confronted by
 and ok claudio bcook

---
 src/usr.sbin/ntpd/ntpd.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/usr.sbin/ntpd/ntpd.c b/src/usr.sbin/ntpd/ntpd.c
index 8106e5fc43f..ccc2c12509b 100644
--- a/src/usr.sbin/ntpd/ntpd.c
+++ b/src/usr.sbin/ntpd/ntpd.c
@@ -1,4 +1,4 @@
-/*	$OpenBSD: ntpd.c,v 1.143 2025/08/20 10:40:21 henning Exp $ */
+/*	$OpenBSD: ntpd.c,v 1.144 2026/04/21 14:36:00 henning Exp $ */
 
 /*
  * Copyright (c) 2003, 2004 Henning Brauer <henning@openbsd.org>
@@ -138,7 +138,7 @@ main(int argc, char *argv[])
 	int			argc0 = argc, logdest;
 	char			**argv0 = argv;
 	char			*pname = NULL;
-	time_t			 settime_deadline;
+	time_t			 settime_deadline = 0;
 	int			 sopt = 0;
 
 	if (strcmp(__progname, "ntpctl") == 0) {
-- 
2.53.0

